### ISSUE TITLE
Thoughts on the "Not a bot?" reporting feature

### ISSUE THREAD

**Issue by beaugunderson** [beaugunderson/bot.community#4]
Should this be changed into two votes, "this is art" and "this isn't art"?

**Comment by mattlaschneider** [comment]
Kind of difficult to say: I love the @big_ben_clock bot, but I imagine a lot of folks would vote "not art" on that one. Does it belong on this site? I would hope so. Although I guess that's an argument for having the two options, at the end of the day.

**Comment by beaugunderson** [comment]
Art/not art is the wrong question, more like "does this bot belong on the site based on the guidelines"? yes/no

**Comment by mattlaschneider** [comment]
yeah, that makes sense, then.

**Comment by hugovk** [comment]
The more important question is  "does this bot belong on the site based on the guidelines?".

But "is it art?" could also be included, as although it clearly has no correct answer might still produce some interesting results.

**Comment by techdubb** [comment]
a snippet from #botally

```
h0ke - beaujingles: w/r/t bot.community - what constitutes an 'art bot'.
beaujingles - h0ke: that is a good question, and i think maybe ‘art bot’ is slightly too narrow
h0ke - is it art, as in visual bots? or art as in, not entirely utilitarian bots (which i guess could be art as well)
beaujingles - maybe it involves ‘content made for the medium’, like a bot made by the BBC that just tweets links based on an RSS feed of new news stories doesn’t seem like it should go on bot.community, it’s just injecting into the medium rather than made for it
h0ke - gotcha.
beaujingles - the text on the page now should probably just say ‘not a bot?’ cuz there are a lot of human ones that slipped through and probably more useful to remove those than trim the actual bots at this point :)
```

tl;dr - this link should probably read 'not a bot?'
